### PR TITLE
Fix scrub-otp-release script for when `.noscrub` isn't used

### PR DESCRIPTION
**Pull request by ConnorRigby**

### CODE REVIEW

**Comment by entone** [comment]
I couldn't help myself.

no scrubs

**Comment by ConnorRigby** [comment]
@entone i've been fighting the urge for 2 days!

**Comment by entone** [comment]
You're a stronger person than I am :)

**Comment by fhunleth** [comment]
@ConnorRigby I reworked the script and added code to exercise it to the smoke test. The smoke test doesn't check the no .noscrub case. My excuse is that I'm lazy and that since that case is so common, it will be hard to miss. I'll manually verify that shortly.

File: tests/smoke-test/rootfs_overlay/srv/erlang/erts-11.1.8/bin/start [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# %CopyrightEnd%
+#
+# This program invokes the erlang emulator by calling run_erl.
+# It should only be used at an embedded target system.
+# It should be modified to give the correct flags to erl (via start_erl),
+# e.g -mode embedded -sname XXX
+#
+# Usage: start [Data]
+#
+ROOTDIR=/home/fhunleth/.asdf/installs/erlang/23.2.7
```
Comment (jjcarstens) [inline tests/smoke-test/rootfs_overlay/srv/erlang/erts-11.1.8/bin/start:28]: Seems like this shouldn't work in CI?

**Review by jjcarstens — APPROVED** [review]

**Comment by jjcarstens** [comment]
Sorry...meant to hit "comment", but actually hit "approve" - though I think it should be okay

File: tests/smoke-test/rootfs_overlay/srv/erlang/erts-11.1.8/bin/start (hunk 1 above)
Comment (fhunleth) [reply tests/smoke-test/rootfs_overlay/srv/erlang/erts-11.1.8/bin/start:28]: Interesting. That file should never be run and should be deleted by the scrubber.

**Pull request merged**